server/test/data/noncommercialTestData.es6

```diff
@@ -163,8 +175,8 @@ module.exports = {
     'type': 'noncommercial',
     'eventName': 'Fun Party',
     'signature': 'LS'
-  },
-  singlePermitHolderOrganizationWithCustomAddresses: {
+  }),
+  singlePermitHolderOrganizationWithCustomAddresses: factory.factory({
```

Comment:
With the factories, I wonder if it makes sense to populate more of the factory and then you can just fill in the field you need for the specific test case.